src/pages/ProductDetail/ProductHeader/ProductHeader.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { FiPlusCircle, FiHeart } from 'react-icons/fi';
+import Logo from '../../../assets/ProductDetail/aesop-logo.png';
+import ProductRelated from './ProductRelated/ProductRelated';
+import './ProductHeader.scss';
+
+function ProductHeader() {
+  const [productDetailList, setProductDetailList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/productDetailList.json')
+      .then(response => response.json())
+      .then(result => setProductDetailList(result));
+  }, []);
+
+  return (
+    <div>
+      {productDetailList.map(productInfo => {
+        return (
```

Comment:
1. 최상위 div에 부여할 스타일링이 없다면 <></> 를 사용해보세요!
2.  아래를 참고하여, 구조분해할당을 진행해주세요
```suggestion
  return (
    <div>
      {productDetailList.map(({id, name, .....})=> {
        return (
```